engine/buildkit/executor.go

```diff
@@ -41,11 +41,14 @@ import (
 )
 
 type ExecutionMetadata struct {
+	ExecID      string
 	ClientID    string
 	SessionID   string
 	SecretToken string
 	Hostname    string
 
+	Internal bool
```

Comment:
nit: ExecID and Internal are a bit less self-evident by their names in terms of what their purpose is, can just add comments for these fields

Reply:
Done!